session6/src/HomeworkS6/HomeworkS6.java

```diff
@@ -0,0 +1,28 @@
+package HomeworkS6;
+
+public class HomeworkS6 {
+    public static void main(String[] args) {
+
+        //Ex:1
+        String stringOne = "OpenAi";
+        String stringTwo = "OpenAi";
+        //System.out.println(stringOne == stringTwo);
```

Comment:
why commented? In this case `stringOne` and `stringTwo` are unused, therefore useless.